chromium_src/chrome/browser/ui/views/frame/browser_view.cc

```diff
@@ -32,14 +33,22 @@
 #define BookmarkBarView BraveBookmarkBarView
 #define UpdateExclusiveAccessBubble UpdateExclusiveAccessBubble_ChromiumImpl
 #define MultiContentsView BraveMultiContentsView
+#define MultiContentsViewDelegateImpl BraveMultiContentsViewDelegateImpl
 
 #define BRAVE_BROWSER_VIEW_LAYOUT_CONVERTED_HIT_TEST \
   if (dst->GetWidget() != src->GetWidget()) {        \
     return false;                                    \
   }
 
+// When web panel's tab is activated, don't need to change
+// tab contents. Tab contents is still same.
+#define BRAVE_BROWSER_VIEW_ON_ACTIVE_TAB_CHANGED \
+  change_tab_contents &= !IsWebPanelContents(new_contents);
```

Comment:
I guess this line also should covers where

```
old_contents == web_panel_contents &&
(new_contents == multi_contents_view_->GetActiveContentsView()->web_contents() ||
 new_contents == multi_contents_view_->GetInactiveContentsView()->web_contents())
```

i.e. active tab is changed from web panel contents to one of visible tabs in contents area.

Reply:
```
bool tab_change_between_contents_view_and_web_panel =
```